website/docs/r/container_cluster.html.markdown

```diff
@@ -344,6 +344,12 @@ addons_config {
 }
 ```
 
+The `database_encryption` block supports:
```

Comment:
Do you mind adding a block like https://github.com/terraform-providers/terraform-provider-google-beta/pull/649/files#diff-6f8c4f3a0d3d3a6b61368f7ab66449e4R327 to describe the top-level field?

Reply:
Sure thing -- just added.